### PR TITLE
[BEAM-4641] Samza runner postcommit status in PR template

### PR DESCRIPTION
**Pull request by xinyuiscool**
**Please** add a meaningful description for your change here

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Spark
--- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by xinyuiscool** [comment]
@iemejia : It makes a lot of sense to keep the lexicographical order. I updated the pr for it. 

For your first comment, it seems to the current way of displaying runners horizontally makes the visualization nicer to me and fits the screen better. It also matches the layout in the capability matrix. If there is no strong opinion, I would like to keep as it is.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ It will help us expedite review of your Pull Request if you tag someone (e.g. `@
...
+Python | [![Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [![Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [![Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/badge/icon)](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---
```
Comment (iemejia) [inline .github/PULL_REQUEST_TEMPLATE.md]: ' | ---' missing here.

**Comment by iemejia** [comment]
Sure, let's see how it looks when merged but well with more runners putting things to the right won't scale.
Let just a minor comment and I will merge once ok.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (xinyuiscool) [reply .github/PULL_REQUEST_TEMPLATE.md]: It's probably really hard to see from this diff, but there is an extra "| ---" in  there :). Previous it has 3 "---" in the end, now we have 4.  I double checked in the html rendering in intellij and it looks ok. Thanks for paying attention!

Comment (iemejia) [reply .github/PULL_REQUEST_TEMPLATE.md]: you are right sorry for the confusion.

**Pull request merged**